aks/postgres/data.tf

```
@@ -35,5 +35,5 @@ data "azurerm_monitor_action_group" "main" {
 data "azurerm_monitor_diagnostic_categories" "main" {
   count = var.azure_enable_backing_services ? 1 : 0
```

```suggestion
  count = local.azure_enable_monitoring ? 1 : 0
```

Should this be using the local as well?